### PR TITLE
Update _Handling failures_ Section of ProActiveUserGuide

### PR DESCRIPTION
**Pull request by sgRomaric**

### CODE REVIEW

File: src/docs/user/ProActiveUserGuide.adoc [hunk 1]
```diff
@@ -1443,27 +1443,26 @@ Here is the list of topologies we support:
...
-restart on an other resource. Defining this property will set the restartTaskOnError property of each task to this
-value as the default one (excepted if this property has been set at the task level). Possible values are 'anywhere'
-or 'elsewhere' meaning respectively that the concerned task will be restart on any available resources or especially
-on a different one. A task can be restarted when an exception occurred (Java Task) or an error code is returned (Native Task).
-
-* *maxNumberOfExecution* (optional - `1` by default) - defines how many times tasks are allowed to be restarted.
-Defining this property will set the nbMaxOfExecution property of each task to this value as the default one
-(excepted if this property has been set at the task level). The value has to be a non-negative and non-null integer.
+It’s possible to have errors when your job is executed. *ProActive Scheduler* provides several mechanisms to deal with exceptional situations in your jobs.
+if a task contains errors during job execution, the *Proactive Scheduler* deals with this situation according to the task error management policies *(TeMP)*
```
Comment (MaelAudren) [inline src/docs/user/ProActiveUserGuide.adoc]: if -> If

File: src/docs/user/ProActiveUserGuide.html [hunk 2]
```diff
@@ -0,0 +1,3480 @@
+<!DOCTYPE html>
```
Comment (MaelAudren) [inline src/docs/user/ProActiveUserGuide.html]: Why did you add ProActiveUserGuide ? If it is an error please remove it. Compiled file should not be added on github.

File: src/docs/user/ProActiveUserGuide.adoc (hunk 1 above)
Comment (sgRomaric) [inline src/docs/user/ProActiveUserGuide.adoc]: Thanks

File: src/docs/user/ProActiveUserGuide.html (hunk 2 above)
Comment (sgRomaric) [inline src/docs/user/ProActiveUserGuide.html]: Yes, this file is auto-generated from a ascidoc file. I will remove it, Thanks Mael

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#Mon May 30 14:05:34 CEST 2016
+#Tue Mar 14 11:56:52 CET 2017
```
Comment (lpellegr) [inline gradle/wrapper/gradle-wrapper.properties]: This change is not required.

File: src/docs/user/ProActiveUserGuide.adoc [hunk 4]
```diff
@@ -1443,27 +1443,24 @@ Here is the list of topologies we support:
...
-
-* *maxNumberOfExecution* (optional - `1` by default) - defines how many times tasks are allowed to be restarted.
-Defining this property will set the nbMaxOfExecution property of each task to this value as the default one
-(excepted if this property has been set at the task level). The value has to be a non-negative and non-null integer.
+It’s possible to have errors when your job is executed. *ProActive Scheduler* provides several mechanisms to deal with exceptional situations in your jobs.
+If a task contains errors during job execution, the *Proactive Scheduler* automatically restarts this task. The number of automatic restarts is defined by the workflow owner by assigning an integer value to the counter *Number of Automatic Restarts*.
+The user also defines *Where he wants that the scheduler restarts In-Error tasks*, i.e., on the same proactive node or on another proactive node.
+
+If automatic restarts does not fix errors, the scheduler applies one of the following task error management policies, i.e., the one defined by the user during workflow creation:
```
Comment (lpellegr) [inline src/docs/user/ProActiveUserGuide.adoc]: Based on README file available at the root of the project:

> Use capitals for ProActive’s terms, as defined in the glossary
> 
> Workflow, Job, Scheduler
> 

scheduler -> Scheduler 

same apply for Task, etc.
